app/views/account/voucher.scala.html

```diff
@@ -28,7 +27,7 @@ <h3 class="mma-section__header">
         <h3 class="mma-section__header">
           Your billing schedule
         </h3>
-        @views.html.account.fragments.billingSchedule(billingSchedule, subscription.currency)
+        @views.html.account.fragments.billingSchedule(billingSchedule, subscription.plan.charges.price.currencies.head)
```

Comment:
The difficulty of safely digging a currency out of a paid subscription plan is quite annoying but its something that can be sorted out in the future I think